packages/graph-engine/src/nodes/array/arraySubgraph.ts

```diff
@@ -42,6 +42,11 @@ export default class ArraySubgraph<T, V> extends Node {
 		const existing = !!props.innerGraph;
 		this._innerGraph = props.innerGraph || new Graph();
 
+		// Automatically propagate the parent’s loader, if any
+		if (props.graph?.externalLoader) {
```

Comment:
Nice, lets just add a test for this . I know it will be simple, but its expected behaviour we want to ensure continues once we do the next major release